catalogue/webapp/pages/concept.tsx

```diff
@@ -213,56 +188,62 @@ export const ConceptPage: NextPage<Props> = ({
                       link={`/images?source.subjects.label=${conceptResponse.label}`}
                     />
                   </Space>
-                </>
+                </div>
               )}
               {((hasImagesTabs && selectedImagesTab === 'images-by') ||
                 (!hasImagesTabs && !!imagesBy?.totalResults)) && (
-                <>
+                <div
+                  role="tabpanel"
+                  id="tabpanel-imagesBy"
+                  aria-labelledby="tab-imagesBy"
+                >
                   <ImageEndpointSearchResults
                     images={imagesBy}
                     background="transparent"
                   />
                   <SeeMoreButton
                     text={`All images (${imagesBy.totalResults})`}
-                    link={`/images?source.subjects.label=${conceptResponse.label}`}
+                    link={`/images?source.contributors.agent.label=${conceptResponse.label}`}
                   />
-                </>
+                </div>
               )}
             </Space>
           </div>
         </ConceptImages>
       )}
 
+      {/* Works */}
       {hasWorks && (
         <>
-          <ConceptWorksHeader
-            as="div"
-            v={{ size: 'xl', properties: ['padding-top'] }}
-            hasWorksTabs={hasWorksTabs}
-          >
+          <ConceptWorksHeader as="div" hasWorksTabs={hasWorksTabs}>
```

Comment:
I think `ConceptWorksHeader` is a styled `Space`, which itself is a `div` so I don't think the `as="div"` is needed.

Reply:
Yeah not sure how that came about 😅 I'll remove